.github/workflows/docker-push.yml

```diff
@@ -0,0 +1,38 @@
+name: Build and Push Docker Image of operator
+
+on:
+  push:
+    tags:
+        - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Create Docker config with auth
+        run: |
+          mkdir -p ~/.docker
+          echo '${{ secrets.DOCKER_AUTH_CONFIG }}' > ~/.docker/config.json
+
+      - name: Extract tag name
+        run: echo "TAG=${GITHUB_REF#refs/tags/tarantool-operator-}" >> $GITHUB_ENV
```

Comment:
I think it's better to use only the version without the prefix in tags. For example just `0.1.0`